dashboard/test/ui/features/teacher_tools/courses_eyes.feature

```diff
@@ -54,7 +54,7 @@ Scenario: Signed out courses, teach
   And I wait to see "#headerlinks"
   And I see "#header-teach"
   And I press "header-teach"
-  Then I am on "https://code.org/teach"
+  Then I am on "http://code.org/teach"
```

Comment:
Using https here seemed to be causing problems on Drone when I was testing. The SauceLabs video looks incorrect when I look at it prior to this change (ie, it's a white page with a TLS error). There are no other examples of '"I am on "https"' in our code base, so I think this is a typo.